assets/generators/webserver/apache/blocks/server.blade.php

```diff
@@ -8,7 +8,7 @@
 
     @if($media)
         # media directory
-        alias "/media/" "{{ $media }}"
+        alias "/media/" "{{ $media . DIRECTORY_SEPARATOR }}"
```

Comment:
There's no real need to use DS here, the alias itself already holds a forward slash, let's stick to that. If you ammend that I'll gladly merge and bring in for 5.1.0

Reply:
But that changes it back to the original state, and this is the only change in this PR... (o.0)

Reply:
@fletch3555 `alias "/media/" "{{ $media "}}/"`

Reply:
AH, see that makes much more sense. 👍